### DISCUSSION TITLE
Training image loading problems

### DISCUSSION THREAD

**Discussion by hhanying** [open-mmlab/mmdetection#11769] [Q&A]
I have 6000 training set data but the actual training data loaded is just over 3000, as shown in the figure batchsize set at 2:
![image](https://github.com/open-mmlab/mmdetection/assets/78523281/ea26fdda-c5cd-4f44-98fe-f8d91fe25cf0)

But my test set of 3000 was able to load them all:
![image](https://github.com/open-mmlab/mmdetection/assets/78523281/c8f95e70-7291-4104-83d2-ad081b958a4a)
How do we fix this?
